doonut-app-external-api/src/main/kotlin/com/doonutmate/member/controller/dto/NameRequest.kt

```diff
@@ -0,0 +1,9 @@
+package com.doonutmate.member.controller.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(title = "회원이름 요청")
+data class NameRequest(
+    @Schema(title = "닉네임")
+    val nickname: String
```

Comment:
프론트에서 1차 검증해서 넘어오니 간단하게 Req 객체 내에서 Validation 걸어줘도 좋을 것 같아요.  

```suggestion
    @Schema(title = "닉네임")
    @jakarta.validation.constraints.NotBlank
    @jakarta.validation.constraints.Size(min = 1, max = 8)
    val nickname: String
```